extendr-api/Cargo.toml

```
@@ -50,5 +52,4 @@ tests-graphics = ["tests-minimal", "graphics"]
 tests-all = ["tests", "graphics"]
 
 [package.metadata.docs.rs]
-features = ["graphics", "either", "ndarray", "num-complex", "serde", "libR-sys/use-bindgen"]
-
+features = ["graphics", "either", "ndarray", "num-complex", "serde"]
```

We need bindgen for the docs to build. Recall #426.
```suggestion
features = ["full-functionality", "libR-sys/use-bindgen"]
```